src/main/docs/guide/installation/installHomebrew.adoc

```diff
@@ -2,13 +2,15 @@ In order to install Micronaut, run following command:
 
 [source,bash]
 ----
-$ brew install --cask micronaut-projects/tap/micronaut
+$ brew install micronaut
```

Comment:
These are not the same.

Your change will make users install Homebrew's own formula, which we don't have control of, and it's not using the native version.

We have our own cask for that

Reply:
Ah, okay, thanks! I tested it and couldn't reproduce the actual issue, so I assumed it was my laptop settings. I'll change it, thanks!

Reply:
I've reverted this change.